docassemble/ALDashboard/data/questions/install_assembly_line.yml

```diff
@@ -198,31 +218,111 @@ fields:
     hide if: skip_efile
     required: False
 ---
-id: feedback form config
+# This must be tested with a private repository
+#  If your GitHub organization's repositories are private, the account must have permission to see it and file issues.
+id: github PAT
 continue button field: feedback_form_config
 question: |
-  GitHub feedback and about information
+  Feedback and "About" page information with GitHub
 subquestion: |
-  You should configure a dedicated GitHub account with permission to make
-  pull requests on public repositories and generate a private access token
-  for it.
+  Assembly Line uses GitHub to let you:
+  
+  1. Get user feedback about each interview.
+  2. Display a "last updated" date on the "About" page of an interview.
+
+  First, in order to use GitHub you should create a dedicated GitHub account. You can use an account that already exists, but we recommend having an account that only handles Assembly Line functionality.
+  
+  The account must have permission make pull requests on your GitHub organization's repositories. You can read [GitHub's documentation about setting an account's permissions](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization).
```

Comment:
From the linked page, it looks like you just need to be able to make issues, with is Read permissions? Am I missing something there?

If you do just need to be able to make issues, I'd say that instead of making PRs, since the account won't actually do that.

Reply:
That's left over from the previous version, though I think that with a private repo it might be true. I think you do need full repo level permissions at least. Not sure if how far that requirement goes. I don't know if I have bandwidth for experimenting at the moment...

Reply:
I would just change it to "full repo level permissions," since we know that works. It's confusing, because permissions to "make pull requests" or "make issues" don't exist when you're making a new PAT, and it's misleading (even to you and I, who know our way around GitHub pretty well!).

Can leave it for now if it's not the focus of this PR though. Feel free to resolve.